docs/Consumer/consumer_data_sharing_agreement.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 6
+---
+
+# Data Sharing Agreement
+The data sharing agreement is a agreement between the AIP and AIU which determines the dataset/resource, list of purposes for data usage and how long the data can be accessed by the AIU.
```

Comment:
The data sharing agreement is an agreement between the AIP and AIU for personal data access. It determines the dataset/resource, list of purposes for data usage, how long the data can be accessed by the AIU and the terms and conditions for data access.